tests/Behat/Service/ElasticsearchCommands.php

```diff
@@ -70,19 +63,21 @@ public function populateAllIndexes(): void
         ];
 
         foreach ($indexes as $index) {
-            $event = new IndexPopulateEvent($index, true, $options);
-            $this->dispatcher->dispatch($event, IndexPopulateEvent::PRE_INDEX_POPULATE);
```

Comment:
Why we change pre populating into post populating?

Reply:
Bump?

Reply:
Changing `PRE_INDEX_POPULATE` into `POST_INDEX_POPULATE` was my oversight. I changed it here: https://github.com/BitBagCommerce/SyliusCrossSellingPlugin/pull/32/commits/a1a2a2902dee5184f65b59f9a1db360dc974b08c

Reply:
Ok, so in the end, we should use POST_INDEX_POPULATE? As it's currently implemented